praetorian_cli/sdk/entities/ad.py

```
@@ -0,0 +1,455 @@
+        :return: A tuple containing (list of relationship results, next page offset)
+        :rtype: tuple
+        """
+        rel_label = self._resolve_relationship(relationship_type) if relationship_type else None
+
+        source_labels = self._resolve_type(source_type) if source_type else None
+        target_labels = self._resolve_type(target_type) if target_type else None
+
+        source_filters = key_equals(source_key) if source_key else None
+        target_filters = key_equals(target_key) if target_key else None
+
+        source_node = Node(labels=source_labels, filters=source_filters)
```

Ruff (0.15.1)

[error] Local variable `source_node` is assigned to but never used

Remove assignment to unused variable `source_node`

(F841)